src/VoyagerServiceProvider.php

```diff
@@ -248,7 +249,7 @@ private function registerPublishableResources()
                 "{$publishablePath}/config/voyager.php" => config_path('voyager.php'),
             ],
             'lang' => [
-                "{$publishablePath}/lang/" => base_path('resources/lang/'),
+                "{$publishablePath}/lang/" => resource_path('lang/vendor/voyager'),
```

Comment:
I like the idea of keeping translations inside the package and namespacing them, however, then we should not allow publishing them.  Much like views, devs can simply copy the file to `resources/lang/vendor/voyager/` on an as-needed basis.  Publishing them all will provide a permanent override and will make future upgrades difficult.  I suggest removing the `lang` entry altogether